src/main/resources/schemas/tekton.dev/v1alpha1_Pipeline.json

```diff
@@ -1253,7 +1253,7 @@
           "type": "array"
         },
         "timeout": {
-          "pattern": "^[-+]?([0-9]*(\\.[0-9]*)?(ns|us|µs|μs|ms|s|m|h))$",
+          "pattern": "^[-+]?([0-9]*(\\.[0-9]*)?(ns|us|µs|μs|ms|s|m|h))+$",
```

Comment:
My only doubt are the first asterisks. In the current form, `1hm2s` is valid or also `1.h`, maybe `^[-+]?([0-9]+(\\.[0-9]+)?(ns|us|µs|μs|ms|s|m|h))+$` is better??

Other "issue" i noticed (which i write so we are aware of it) is the order of the units - 10m1h is accepted. But i guess we should trust users for this.

Reply:
According to the Go code, both are valid:

- .1s or 1.s are valid
- no ordering for units